runtime-sdk/modules/evm/src/types.rs

```diff
@@ -0,0 +1,168 @@
+//! EVM module types.
+#[derive(Clone, Debug, cbor::Encode, cbor::Decode)]
+pub struct CreateTx {
+    pub value: Vec<u8>, // U256
```

Comment:
can you not make it a `U256` directly? it's just a `[u8; N]`. It'll save you a few copies (or unsafe code) if it's direct.